EIPS/eip-4345.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: 4345
+title: Difficulty Bomb Delay to April 2022
+author: Tim Beiko (@timbeiko)
+discussions-to: TBA
+status: Final
+type: Standards Track
+category: Core
+created: 2021-10-05
+---
+
+## Simple Summary
+Delays the difficulty bomb to show effect in April 2022.
```

Comment:
This section has been deprecated in favor of the `description` preamble field. Please see the `eip-template.md` to see how it's used.

Reply:
Fixed, thanks!